src/features/embalm/stepContent/hooks/useArchaeologistList.ts

```diff
@@ -1,16 +1,20 @@
-import { useCallback } from 'react';
+import { useCallback, useMemo } from 'react';
 import {
   deselectArchaeologist,
   selectArchaeologist,
   SortDirection,
   setDiggingFeesSortDirection,
+  setUnwrapsSortDirection,
```

Comment:
I think these sort and filter properties should be moved to their own `store`. It seems we have bloated the `embalm` store. Going forward `embalm` should only hold values required for creating a sarcophgus. And then UX and other properties can be in either their own store, or in this case maybe something like `archaeologistList`. You should be able to follow the pattern for the other `stores` but ask if you have any questions

Reply:
I created its own store -- @mobiusTripper-crypto what did you mean with archaeologistList?